app/views/admin/admins/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="container">
+  <h3 class="text-center">Admin Dashboard</h3>
+  <%= link_to "Add Product", new_admin_product_path, class: "btn btn-primary" %>
```

Comment:
Reading the user story for this, it sounds like it would be a better UX to have the product listing and the product creation piece on the main admin/dashboard page.
This should be similar to how comments were embedded on the article show page in blogger.

What are your thoughts?